cfgov/unprocessed/js/molecules/Expandable.js

```diff
@@ -370,6 +371,20 @@ function _getTransitionEndEvent( elm ) {
 }
 
 /**
+ * @param {string} transitionEnd The browser-prefixed transition end event.
+ * @returns {string} The browser-prefixed transition event.
+ */
+function _getTransitionPrefix( transitionEnd ) {
+  var TRANSITION_PREFIXES = {
+    transitionend      : 'transition',
+    webkitTransitionEnd: '-webkit-transition'
```

Comment:
Do you need the other platforms prefixed here as well? https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/js/modules/transition/BaseTransition.js#L167-L172